Sources/Testing/Running/SkipInfo.swift

```diff
@@ -34,10 +34,11 @@ public struct SkipInfo: Sendable {
   ///     Defaults to `nil`.
   ///   - sourceContext: A source context indicating where this skip occurred.
   ///     Defaults to a source context returned by calling
-  ///     ``SourceContext/init(backtrace:sourceLocation:)`` with zero arguments.
+  ///     ``SourceContext/init(backtrace:sourceLocation:)`` passing only the
+  ///     current backtrace.
   public init(
     comment: Comment? = nil,
-    sourceContext: SourceContext = .init()
+    sourceContext: SourceContext = .init(backtrace: .current())
```

Comment:
This will still capture the wrong backtrace.

Reply:
Discussed offline; this one _already_ contained undesired frames in the backtrace. This change should reduce the number of junk frames, improving things slightly. But I'm actually going to change this further to remove the default `SourceContext` from `SkipInfo.init` entirely.

Reply:
We'll need to keep a deprecated source-compatible initializer around since this is tools SPI.